### DISCUSSION TITLE
multilockon123氏によって更新されたタイルを標準搭載してほしい

### DISCUSSION THREAD

**Discussion by pacchinot** [hengband/hengband#3638] [Ideas]
タイル使ってる人って割と多いと思うんです

**Comment by daradarach** [comment]
既に標準搭載されてるけど最新alphaはバグでタイル表示ができないんじゃありませんでしたっけ。
それもバグ自体は修正済みでリリース待ちなので次で直るとか

**Reply by pacchinot** [reply]
今の開発版に標準でついてるのは未完成なタイルで敵のタイルに欠けがあったりするんです
multilockon123氏の更新されたタイルではそれが修正されているんですが標準タイルは更新されていないんです
